config.cfg

```diff
@@ -0,0 +1,52 @@
+# Copyright 2016-2016 the openage authors. See copying.md for legal info.
+
+set TOGGLE_CONSOLE `
```

Comment:
how about using `cfg/keybinds.oac` then? that cfg folder is installed to `/usr/share` as a template to initialize the `~/.config/openage/cfg` folder then.